Dashboard documentation refactor (#116)
SolarArbiter/SolarArbiter.github.io: dashboard_getting_started.md

administrators at [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org)
to start the process. The framework administrators will provide the
non-negotiable [Data Use Agreement](/assets/45864%20Approved_Final%20version%201.1.pdf),
the user will review the Data Use Agreement and return the Data Use Agreement
signed by an authorized representative of their organization. A [point of contact](#organization-point-of-contact) will be established to verify future updates to the
organization. An organization administrator will be appointed by the point of
contact to manage data access.

#### Joining an Organization
{: .anchor}

Users can join an existing organization by emailing the framework
administrators at [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org).
Framework administrators will verify with the organization's point of contact
that the user is authorized to join the organization. To expedite this process,
an organization's point of contact may initiate a request to add the user to
their organization.

### User Classifications
{: .anchor}

Users of the Solar Forecast Arbiter will fall into one of the following

> classifications.

Reviewer: Users of the Solar Forecast Arbiter are described by one or more of the following classifications.

#### Organization Point of Contact
{:.anchor}

An organization's point of contact is a representative of the organization who
is authorized to approve or request changes to the organization. There may be
more than one point of contact for an organization. A point of contact will be
capable of contacting the framework administrators to complete the following

> actions:

Reviewer: A point of contact may coordinate with framework administrators to:

> - Adding and removing users from the organization.

Reviewer: ```suggestion
- Add or remove users from the organization.
```

> - Promoting users to [organization administrators](#organization-administrators).

Reviewer: ```suggestion
- Promote users to [organization administrators](#organization-administrators).
```

> - Participating in trials.

Reviewer: Coordinate organization participation in a forecast trial.

#### Organization Administrators
{: .anchor}

> Organization Administrators are users with special permissions to act on users,

Reviewer: need a link to role based access control docs. maybe multiple links in this section.

roles and permissions in their organization. Organization administrators are

> also granted full access to their organization's data. An organization's point

Reviewer: isn't this true for all users by default? if so, let's remove to avoid confusion

Author: When a user joins an organization, they don't gain any additional permissions until an administrator grants them. The only thing that changes in the transaction is the user's organization.

Reviewer: ok, now I remember discussing this a few weeks ago and forgot.

of contact may request that users in their organization be promoted to an
organization administrator by emailing a request to
[admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org).

> Administrators will be allowed to [grant](/documentation/dashboard/administration#granting-roles-to-a-user)

Reviewer: ```suggestion
Administrators may [grant](/documentation/dashboard/administration#granting-roles-to-a-user)
```

and [revoke](/documentation/dashboard/administration#revoking-roles-from-a-user)
roles from users in their organization and add or remove permissions from
roles. Roles may be granted to members of other organizations provided their
organization has aggreed to the Data Use Aggreement.

All administrative permissions other than read access, e.g. those allowing
create, update, grant, revoke or delete on users, roles or permissions, are
restricted to sharing with users in the same organization. For example,
attempts to grant a role with permission to *update users* to a member of
another organization or to add the *update users* permission to a role that
has been granted outside the organization will not be allowed.
        

> #### Standard user

Reviewer: ```suggestion
#### Standard User
```

{: .anchor}

Solar Forecast Arbiter users are added to the *Unaffiliated* organization by
default. Unaffiliated users will have access to the reference data set but
may not gain further access until they have joined an organization that has
accepted the data use agreement.

An organization's [point of contact](#organization-point-of-contact) may submit
a request to [admin@solarforecastarbiter.org](mailto:admin@solarforecastarbiter.org)
to add a user to their organization. Once the user has been added to the
organization, an [organization administrator](#organization-administrators) may

> grant the user access to the organization's data.

Reviewer: same confusion about default read permissions.

Author: would it clarify things to add something like 

Users new to an organization have no access to the organization's data until an [organization administrator](#organization-administrators) grants them access.

Reviewer: yes
